src/web3/createMulticallContractInstance.ts

```diff
@@ -0,0 +1,35 @@
+import MulticallAtrifact from "@uma/core/build/contracts/MulticallMock.json";
```

Comment:
nit: probably slightly preferable to import the interface `Multicall` rather than `MulticallMock` (the implementation) since you shouldn't need the full implementation to know how to interact with it.